alcs-api/config/default.json

```diff
@@ -8,7 +8,7 @@
   "DATABASE_USER": "postgres",
   "DATABASE_PASSWORD": "postgres",
   "DATABASE_MIGRATION_TABLE": "migrations",
-  "LOG_LEVEL": "warning",
+  "LOG_LEVEL": "warn",
```

Comment:
Note that this will have to be updated on each environments config map in order to be applied in a deployed environment